Gemfile

```diff
@@ -13,4 +13,10 @@ gem "rubocop", "~> 1.7"
 
 gem "rubocop-rake"
 
-gem "rubocop-rspec"
\ No newline at end of file
+gem "rubocop-rspec"
+
+gem "webmock", "~> 2.1"
+
+gem "byebug", "~> 11.1.3"
+
+gem "mock_redis", ">=0.14"
```

Comment:
I think these were dev dependencies in muffin_man.gemspec. Do we need to keep this as required dependencies?